examples/AsyncSummarizeMediaUrl.java

```diff
@@ -0,0 +1,113 @@
+package ai.rev;
+
+import ai.rev.speechtotext.ApiClient;
+import ai.rev.speechtotext.models.NlpModel;
+import ai.rev.speechtotext.models.asynchronous.*;
+import ai.rev.speechtotext.models.vocabulary.CustomVocabulary;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.Map;
+
+public class AsyncSummarizeMediaUrl {
+
+  public static void main(String[] args) {
+    // Assign your access token to a String
+    String accessToken = "<YOUR_ACCESS_TOKEN>";
+
+    // Initialize the ApiClient with your access token
+    ApiClient apiClient = new ApiClient(accessToken);
+
+    // Set up source configuration parameters
+    String mediaUrl = "https://www.rev.ai/FTC_Sample_1.mp3";
+
+    // Initialize the RevAiJobOptions object and assign
+    RevAiJobOptions revAiJobOptions = new RevAiJobOptions();
+    revAiJobOptions.setSourceConfig(mediaUrl, null);
+    revAiJobOptions.setDeleteAfterSeconds(2592000); // 30 days in seconds
+    revAiJobOptions.setLanguage("en");
+    revAiJobOptions.setSummarizationOptions(new SummarizationOptions().setModel(NlpModel.STANDARD));
+
+    RevAiJob submittedJob;
+
+    try {
+      // Submit job with transcription options
+      submittedJob = apiClient.submitJobUrl(revAiJobOptions);
+    } catch (IOException e) {
+      throw new RuntimeException("Failed to submit url [" + mediaUrl + "] " + e.getMessage());
+    }
+    String jobId = submittedJob.getJobId();
+    System.out.println("Job Id: " + jobId);
+    System.out.println("Job Status: " + submittedJob.getJobStatus());
+    System.out.println("Created On: " + submittedJob.getCreatedOn());
+
+    // Waits 5 seconds between each status check to see if job is complete
+    boolean isJobComplete = false;
+    while (!isJobComplete) {
```

Comment:
once this is a summarization example, not sure we need this block as it makes the example more complicated.
can we do a summarization polling only?

Reply:
@alexsku are you okay with that?
You were requesting polling those jobs in order in python sdk...